alts/worker/runners/base.py

```diff
@@ -1479,14 +1479,7 @@ def write_to_file(file_base_name: str, artifacts_section: dict):
         except UploadError as e:
             raise PublishArtifactsError from e
 
-    # After: install_package and run_tests
-    @command_decorator(
-        'stop_environment',
-        'Cannot destroy environment',
-        exception_class=StopEnvironmentError,
-        is_abortable=False,
-    )
-    def stop_env(self):
+    def _stop_env(self):
         if os.path.exists(self._work_dir):
```

Comment:
We can decrease nesting in the following way:
```python
if not os.path.exists(self._work_dir):
     return (0, '', '')
```
Also, I see that we're expecting a tuple as the return value in child methods, but we can return None if work_dir doesn't exist.
I think it's impossible, but let's return some default values